src/css/modules/steps.css

```
@@ -0,0 +1,168 @@
+  --lay-steps-wait-color: var(--lay-gray-400); /* 待处理步骤色 */
+  --lay-steps-line-color: var(--lay-gray-300); /* 连接线颜色 */
+  --lay-steps-status-success: var(
+    --lay-color-green
+  ); /* 当前步骤状态色：success */
+  --lay-steps-status-error: var(--lay-color-red); /* 当前步骤状态色：error */
+  --lay-steps-status-warning: var(
+    --lay-color-orange
+  ); /* 当前步骤状态色：warning */
+  --lay-steps-icon-size: 28px; /* 图标尺寸 */
+  --lay-steps-font-size: 14px; /* 图标内文字大小 */
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)